bc_obps/registration/api/user_operator.py

```diff
@@ -306,72 +308,6 @@ def create_operator_and_user_operator(request, payload: UserOperatorOperatorIn):
         return 400, {"message": str(e)}
 
 
-@router.post("/user-operator/contact", response={200: SelectOperatorIn, codes_4xx: Message})
```

Comment:
I removed this instead of leaving it in in case we need it later because I think the contact-adding logic should be in a put on the user-operator endpoint